.github/dependabot.yml

```diff
@@ -1,13 +1,22 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: weekly
-    day: wednesday
-    time: "05:00"
-    timezone: America/New_York
-  open-pull-requests-limit: 99
-  allow:
-  - dependency-type: direct
-  - dependency-type: indirect
+  - package-ecosystem: pip
+    directory: '/'
+    schedule:
+      interval: weekly
+      day: wednesday
+      time: '05:00'
+      timezone: America/New_York
+    open-pull-requests-limit: 99
+    allow:
+      - dependency-type: direct
+      - dependency-type: indirect
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
+      day: wednesday
+      time: '05:00'
+      timezone: America/New_York
+    open-pull-requests-limit: 99
+    versioning-strategy: increase
```

Comment:
Is there a way to only enable these for one of our github organizations so that we aren't having to close extra PRs in the other?

Reply:
Sigh, I wish. It's annoying, I entirely agree. https://github.com/dependabot/dependabot-core/issues/2198#issuecomment-649444075